src/python/bot/fuzzers/engine_common.py

```diff
@@ -312,7 +312,8 @@ def find_fuzzer_path(build_directory, fuzzer_name, is_blackbox=False):
     # Fuchsia targets are not on disk.
     return fuzzer_name
 
-  if environment.platform() == 'ANDROID_KERNEL':
+  if (environment.platform() == 'ANDROID_KERNEL' or
```

Comment:
Better to compare to 'ANDROID_KERNEL_X86' instead of `environment.is_kernel_fuzzer_job()` ? 

Should also create a environment.is_android_kernel() which does this and call that here instead.

Reply:
Added environment.is_android_kernel() that also check QUEUE_OVERRIDE because there will be a case where os_override=android_x86 but running kernel jobs.